tests/engines/registries/integration/components/page-link/component-test.ts

```diff
@@ -0,0 +1,133 @@
+import Service from '@ember/service';
+import { render } from '@ember/test-helpers';
+import { TestContext } from 'ember-test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import { module, test } from 'qunit';
+
+import { setupEngineRenderingTest } from 'ember-osf-web/tests/helpers/engines';
+
+class RouterStub extends Service {
+    urlFor() {
+        return 'http://example.com';
+    }
+
+    isActive() {
+        return false;
+    }
+
+    transitionTo() {
+        // empty
+    }
+}
+
+class CurrentUserStub extends Service {
+}
+
+module('Registries | Integration | Component | page-link', hooks => {
+    setupEngineRenderingTest(hooks, 'registries');
+
+    hooks.beforeEach(function(this: TestContext) {
+        this.owner.register('service:router', RouterStub);
+        this.owner.register('service:current-user', CurrentUserStub);
+    });
+
+    test('Renders an unvisited page link', async function(this: TestContext, assert) {
+        const pageManager = {
+            isVisited: false,
+            pageIsValid: false,
+            pageHeadingText: 'Unvisited',
+        };
+
+        this.setProperties({ pageManager });
+
+        await render(hbs`
+            <PageLink
+                @link={{component 'osf-layout/registries-side-nav/x-link'}}
+                @draftId='test'
+                @pageManager={{this.pageManager}}
+                @pageIndex={{100}}
+                @currentPage={{1}}
+                as |pageLink|
+            />
+        `);
+
+        assert.dom('[data-test-link="101-unvisited"]').exists('Unvisited PageLink Component renders');
+        assert.dom('[data-test-label]').containsText(pageManager.pageHeadingText, 'has proper label');
+        assert.dom('i[data-test-icon].fa-circle').exists('icon uses proper symbol');
```

Comment:
Does the `i` need to be here?  It doesn't look like the other tests are using it in the data-test lookup.